src/components/course/routes/tests/ExternalCourseEnrollment.test.jsx

```diff
@@ -40,6 +40,10 @@ jest.mock('@edx/frontend-platform/config', () => ({
   })),
 }));
 
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+}));
```

Comment:
Is this mock needed? It doesn't look like anything from `react-router-dom` is actually getting mocked (which makes sense given the use of `renderWithRouterProvider`.

Reply:
oh whoops, leftover from a previous iteration